client/src/components/RatingsAndReviews/RatingsAndReviews.jsx

```diff
@@ -2,8 +2,9 @@ import React, { useEffect, useState } from 'react';
 import StarReview from '../StarReview.jsx';
 import ReviewsList from './ReviewsList.jsx';
 import calculateRating from '../../scripts/calculateRating.js';
+import AddReviewModal from './AddReviewModal.jsx';
 
-const RatingsAndReviews = function({ onChangeFileHandler, onClickUploadHandler, productId, reviews, reviewsMeta }) {
+const RatingsAndReviews = function({top, onChangeFileHandler, onClickUploadHandler, productId, reviews, reviewsMeta }) {
```

Comment:
more NITS, you can ignore these if you want
```suggestion
const RatingsAndReviews = function({ top, onChangeFileHandler, onClickUploadHandler, productId, reviews, reviewsMeta }) {
```